stellenbosch-university-department-of-geography-and-environmental-studies-reference-style

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Stellenbosch University Department of Geography and Environmental Studies Reference Style</title>
+    <title-short>SUDGES</title-short>
+    <id>http://www.zotero.org/styles/stellenbosch-university-department-of-geography-and-environmental-studies-reference-style</id>
+    <link rel="self" href="http://www.zotero.org/styles/stellenbosch-university-department-of-geography-and-environmental-studies-reference-style"/>
+    <author>
+      <name>Ockert Malan</name>
+      <email>19475446@sun.ac.za</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <summary>A CSL style in accordance with SU DGES reference guidelines</summary>
+    <published>01-06-2017</published>
+    <updated>2017-07-07T07:58:22+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <link href="https://stellenbosch-my.sharepoint.com/personal/19475446_sun_ac_za/_layouts/15/guestaccess.aspx?docid=0044a91e96c94404ebe0c42d85021a6c6&amp;authkey=ATwYPE-qyPqiTWUmGI_obXk"/>
```

Comment:
@OGMalan, this documentation link doesn't work for me. Do you know if there is a public copy of the guidelines available. Or, alternatively, could you add the file as an attachment to the pull request?